src/azure/cli/commands/arm.py

```diff
@@ -310,6 +310,7 @@ def set_properties(instance, expression):
         elif isinstance(instance, dict):
             instance[name] = value
         else:
+            # TODO: Why not throw an error? setting banana=foo will silently fail
```

Comment:
>Why not throw an error? setting banana=foo will silently fail [](start = 20, length = 61)

Oh yes, this! Question for @BurtBiel or @johanste. Right now it will tell you if you say "somethingReal.somethingFake.foo=bar" that somethingFake can't be found. But if you say "somethingReal.somethingFake=blah" then it will happily do what you say and just fail silently. I could imagine that something legitimate would require this behavior, but if not I'd rather throw an error.